src/coreclr/nativeaot/System.Private.CoreLib/src/System/Runtime/InteropServices/NativeLibrary.NativeAot.cs

```diff
@@ -126,30 +126,19 @@ private static IntPtr LoadLibraryHelper(string libraryName, int flags, ref LoadL
             }
 
             int lastError = Marshal.GetLastWin32Error();
-            if (lastError != LoadLibErrorTracker.ERROR_INVALID_PARAMETER)
+            if (lastError != Interop.Errors.ERROR_INVALID_PARAMETER)
             {
                 errorTracker.TrackErrorCode(lastError);
             }
 
             return ret;
 #else
-            IntPtr ret = IntPtr.Zero;
-            if (libraryName == null)
-            {
-                errorTracker.TrackErrorCode(LoadLibErrorTracker.ERROR_MOD_NOT_FOUND);
-            }
-            else if (libraryName == string.Empty)
+            // TODO: FileDosToUnixPathA
+            IntPtr ret = Interop.Sys.LoadLibrary(libraryName);
```

Comment:
Going by https://github.com/dotnet/runtime/blob/5d3288d449ed61fa1e9da3fe876400ac14c47fa7/src/coreclr/pal/src/file/path.cpp#L449-L451 fix for this TODO is:
```suggestion
            IntPtr ret = Interop.Sys.LoadLibrary(libraryName.Replace('\\', '/'));
```